tsconfig.json

```diff
@@ -67,6 +67,7 @@
 			"utils/*": ["./utils/*"],
 			"facets": ["./facets"],
 			"facets/*": ["./facets/*"]
+			//		"sequelize": ["./types/test.d.ts"]
```

Comment:
Should this just be removed?

Reply:
yes, this was for the type inference things